tox.ini

```diff
@@ -23,7 +23,7 @@ deps =
 
 [testenv:docs]
 basepython =
-    python2.7
+    python3.6
```

Comment:
I admit that I don't quite follow why you're moving coverage to Python 3.7 and docs to Python 3.6.

Reply:
I did that because Sphinx doesn't support 3.7 now. It might not break, I just didn't know how good the support is now.